### ISSUE TITLE
Support filenames/paths with blanks

### ISSUE THREAD

**Issue by jghauser** [jghauser/follow-md-links.nvim#3]

**Comment by AmalIrfan** [comment]
I am not sure but I think in \`common md\` syntax
this
```md
[Link To A File](<Path To The File.md>)
```
works in some places but is unsupported in others.
Right now I use this (amazing) plugin by compromising and naming all files with underscores separating words instead of spaces.
```md
[Link To A File](Path_To_The_File.md)
```
not a big deal, though the former is prettier.

**Comment by tvanyo** [comment]
I would love to have this issue resolved. If I knew where to do it in the code I would make the change.

**Comment by jghauser** [comment]
Implemented by #19
